### PR TITLE
[16.0][IMP] despatch_advice_import_ubl : collect package (logistic transport units)

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

**Review by santostelmo — APPROVED** [review]

**Review by jbaudoux — APPROVED** [review]

File: despatch_advice_import_ubl/wizard/despatch_advice_import.py [hunk 1]
```diff
@@ -125,6 +129,25 @@ def parse_ubl_despatch_advice_line(self, line, ns):
             "uom": {"unece_code": qty_xpath[0].attrib.get("unitCode")},
             "backorder_qty": backorder_qty,
         }
+
+        package_id_xpath = line.xpath(
+            "cac:Shipment/cac:TransportHandlingUnit/cac:ActualPackage/cbc:ID",
+            namespaces=ns,
+        )
```
Comment (simahawk) [inline despatch_advice_import_ubl/wizard/despatch_advice_import.py]: these are elements, not xpaths 
```suggestion
        package_id_el = line.xpath(
            "cac:Shipment/cac:TransportHandlingUnit/cac:ActualPackage/cbc:ID",
            namespaces=ns,
        )
```

Comment (QuocDuong1306) [reply despatch_advice_import_ubl/wizard/despatch_advice_import.py]: Hi @simahawk , I added 1 commit to replace that

**Review by cyrilmanuel — APPROVED** [review]

**Comment by cyrilmanuel** [comment]
hi @simahawk could you merge this PR ?

**Comment by cyrilmanuel** [comment]
Hiii @simahawk all good for you ?

**Review by simahawk — APPROVED** [review]

**Pull request merged**